### PR TITLE
Fix configuration of PLLQ and PLLP

### PR DESCRIPTION
**Pull request by davidlattimore**
The pllqend and pllpen bits weren't being set. Also, the value written
to pllsyscfgr was being overwritten by the subsequent write call.

### CODE REVIEW

File: src/rcc/mod.rs [hunk 1]
```diff
@@ -228,37 +228,30 @@ impl Rcc {
...
+        let mut p = None;
+
+        self.pllsyscfgr.write(|w| unsafe {
+            w.pllsrc().bits(pll_sw_bits);
+            w.pllm().bits(pll_cfg.m - 1);
+            w.plln().bits(pll_cfg.n);
+            w.pllr().bits(pll_cfg.r - 1);
+            w.pllren().set_bit();
+            if let Some(div) = pll_cfg.q {
+                assert!(div > 1 && div <= 8);
```
Comment (andresv) [inline src/rcc/mod.rs:241]: @dotcypress we haven't used asserts so far, what is your take on these?
I vaguely remember that they might increase flash size quite significantly.

Comment (davidlattimore) [reply src/rcc/mod.rs:241]: There's a couple of asserts already at the start of the method. I'd expect that if you're building with LTO and using constants values in your PllConfig, the compiler can probably optimize the asserts away. That said, I'd be happy to change these back to if-statements like they were in the old code. It just seemed odd to me that if you supplied an invalid config, it just ignored the bits that were invalid and kept going.

Comment (dotcypress) [reply src/rcc/mod.rs:241]: @davidlattimore totally agree, this approach is much better

**Review by dotcypress — APPROVED** [review]
LGTM

**Pull request merged**